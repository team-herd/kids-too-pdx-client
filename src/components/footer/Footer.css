.Footer {
  bottom: 0;
  padding: 1rem;
  background-color: #efefef;
  text-align: center;
  font-size: 1.4em;
  margin: 0;
  a {
    color: black;
    text-decoration: none;
  }
  a:hover {
    background-color: lightgray;
    cursor: pointer;
  }
}
