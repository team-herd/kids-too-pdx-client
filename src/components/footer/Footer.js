import React from 'react';
import styles from './Footer.css';

function Footer() {
  return (
    <main className={styles.Footer}>
      <a href='/aboutus'>About Us</a>
    </main>
  );
}
export default Footer;
