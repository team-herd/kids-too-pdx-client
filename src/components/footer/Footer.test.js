import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  it('matches snapshot for Footer', () => {
    const tree = renderer.create(
      <Footer />
    );

    expect(tree).toMatchSnapshot();
  });
});
