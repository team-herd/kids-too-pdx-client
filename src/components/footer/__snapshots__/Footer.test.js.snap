// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot for Footer 1`] = `
<section
  className="Footer"
>
  <a
    href="/aboutus"
  >
    About Us
  </a>
</section>
`;
